models/wsModels/WSConstants.php

```diff
@@ -15,21 +15,37 @@
 namespace app\models\wsModels;
 
 class WSConstants {
+    
+    // result
+    const RESULT = 'result';
 
+    // token
+    const TOKEN = 'token';
+    const TOKEN_INVALID = 'Invalid token';
     const ACCESS_TOKEN = 'access_token';
     const TOKEN_EXPIRES_IN = 'expires_in';
     const TOKEN_COOKIE_TIMEOUT = 'tokenTimeout';
-    const DATA = 'data';
+    
+    // matadata
     const METADATA = 'metadata';
+    const EXCEPTION = 'exception';
+    const DETAILS = 'details';
+    const DATA_FILES = 'datafiles';
+    const LANG = "language";
+    const STATUS = "status";
+    
+    // pagination
     const PAGINATION = 'pagination';
     const TOTAL_PAGES = 'totalPages';
     const TOTAL_COUNT = 'totalCount';
     const CURRENT_PAGE = 'currentPage';
     const PAGE_SIZE = 'pageSize';
     const PAGE = "page";
-    const TOKEN = 'Invalid token';
```

Comment:
the previous value of TOKEN was 'Invalid token';

Reply:
ok